tools/ci_build/github/azure-pipelines/binary-size-checks-pipeline.yml

```diff
@@ -23,6 +23,17 @@ jobs:
     clean: all
   pool: Linux-CPU-2019
 
+  strategy:
+    matrix:
+      AndroidMinimalBaseline:
+        ConfigId: minimal-baseline
+        BuildConfigFile: "/onnxruntime_src/tools/ci_build/github/linux/ort_minimal/build_check_binsize_config/android_minimal_baseline.config"
+        BinarySizeThresholdInBytes: 1306224
+      AndroidMinimalBasicWithSomeOps:
+        ConfigId: android-minimal-basic-with-some-ops
+        BuildConfigFile: "/onnxruntime_src/tools/ci_build/github/linux/ort_minimal/build_check_binsize_config/android_minimal_basic_with_some_ops.config"
+        BinarySizeThresholdInBytes: 2658304
+
```

Comment:
Is it possible to add a variant for the specific 1P usage that is most sensitive to binary size?

Reply:
that would be good. @joburkho any thoughts about what this configuration should be?

Reply:
@edgchen1 : Our new reduced builds on Android and iOS will start to include the sentencepiece operator and a few others in addition to our previous operators.  Once I get good builds on mobile, I'll share the new configuration file(s) and associated build sizes.